src/Periphery/TokenWrapper.sol

```diff
@@ -0,0 +1,65 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.17;
+
+import { LibAsset } from "../Libraries/LibAsset.sol";
+import { TransferrableOwnership } from "../Helpers/TransferrableOwnership.sol";
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+/// External wrapper interface
+interface IWrapper {
+    function deposit() external payable;
+    function withdraw(uint wad) external;
+}
+
+/// @title TokenWrapper
+/// @author LI.FI (https://li.fi)
+/// @notice Provides functionality for wrapping and unwrapping tokens
+/// @custom:version 1.0.0
+contract TokenWrapper is TransferrableOwnership {
```

Comment:
Why is this contract ownable? I don't see any call an owner could do.

Reply:
Other periphery contracts are ownable - I assume is to avoid re-deploy if we change diamond? I can remove it's not really needed here.